config/routes.rb

```diff
@@ -289,6 +292,11 @@
   get "/message/new/:display_name" => "messages#new", :as => "new_message"
   get "/message/read/:message_id", :to => redirect(:path => "/messages/%{message_id}")
 
+  # muting users
+  post "/user/:display_name/mute" => "user_mutes#create", :as => :mute_user
+  delete "/user/:display_name/mute" => "user_mutes#destroy", :as => :unmute_user
```

Comment:
If these can be added as resourceful routes then that would be better. Or at lest, specific actions on a resource or scoped resource. Something like (untested)

```ruby
scope "/user/:display_name" do
  resources :user_mutes, :only -> [:create, :destroy]
end
```

We're (slowly) getting rid of all the "full handwritten" routes so I'm keen to avoid adding more.